### PR TITLE
Update pkgconfig flags for all plugins to include irregular locations of headers on distros like openSUSE

### PR DESCRIPTION
**Pull request by zebogan**
like #64 but i changed the flags for all the plugins. solves https://github.com/hyprwm/Hyprland/issues/5616

### CODE REVIEW

File: hyprbars/Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 CXXFLAGS = -shared -fPIC --no-gnu-unique -g -std=c++2b -Wno-c++11-narrowing
-INCLUDES = `pkg-config --cflags pixman-1 libdrm hyprland pangocairo libinput libudev wayland-server hyprland`
```
Comment (fufexan) [inline hyprbars/Makefile:2]: Is removing `hyprland` intended?

Comment (zebogan) [reply hyprbars/Makefile:2]: correct me if i'm wrong, but it said hyprland twice so i assumed that that was a mistake

Comment (zebogan) [reply hyprbars/Makefile:2]: @fufexan

Comment (fufexan) [reply hyprbars/Makefile:2]: Ah, missed that last night. My bad.

**Review by fufexan — APPROVED** [review]
LGTM, thanks!

**Pull request merged**